plugins/generator-1.19.2/forge-1.19.2/templates/elementinits/blocks.java.ftl

```diff
@@ -36,11 +36,22 @@
 
 package ${package}.init;
 
+<#assign hasTransparentBlocks = false>

```

Comment:
Transparency is specified in the block model, so this variable is no longer needed